proto/cachepubsub.proto

```diff
@@ -118,10 +118,10 @@ message _TopicItem {
   // Authenticated id from Publisher's disposable token
   string publisher_id = 3;
 
-  // Sequence stamps exist to determine which sequence number range a message belongs to. On a topic reset,
-  // the sequence numbers reset and a new sequence_stamp is given.
-  // For a given sequence_stamp, the next message in a topic is topic_sequence_number + 1.
-  uint64 sequence_stamp = 4;
+  // Sequence pages exist to determine which sequence number range a message belongs to. On a topic reset,
+  // the sequence numbers reset and a new sequence_page is given.
+  // For a given sequence_page, the next message in a topic is topic_sequence_number + 1.
```

Comment:
And there is no expectation the sequence pages are sequential, right? If correct, we should document here as the some could assume that from the word `page`.

Reply:
Addressed in https://github.com/momentohq/client-protos/commit/f8491912ef2843561e58ab19b8af0d7f829122ef

Reply:
@malandis is this resolved to your satisfaction?